### DISCUSSION TITLE
MySQL AWS RDS [Aug 06]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3142] [Discord Threads]
> 🙋‍♂️ **Priyanka**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Would need more details, do you mean this is via GUI ? or you are setting in NC_DB ?  
#  
> 🙋‍♂️ **Priyanka**  
#### We are setting it via NC_DB by creating a new application  
#  
> 💁 **o1lab**  
#### can you connect via simple nodejs file via knex   
#### to your mysql rds.. if you can make that work.. this could be worked out  
#  
> 🙋‍♂️ **Priyanka**  
#### Sorry my bad. We are setting it via GUI by creating a new application  
#  
> 🙋‍♂️ **ioakshay**  
#### Hi Olab, would you mind to connect over a short call  
#  
> 💁 **o1lab**  
#### if you can make it work with knexjs before  
#  
> 💁 **o1lab**  
#### I can help you  
#  
> 💁 **o1lab**  
#### saying this cause, I wouldnt know all details of aws rds settings  
#  
> 💁 **o1lab**  
#### http://knexjs.org/guide/#configuration-options  
#  
> 🙋‍♂️ **ioakshay**  
#### Can you please share your email address  
#  
> 🙋‍♂️ **ioakshay**  
#### We are in cal with AWS RDS support they are trying to connect with NocoDB but did not get would you mind  to join the call and suggest us  
#  
> 💁 **o1lab**  
#### 🙂  
#  
> 💁 **o1lab**  
#### sorry I am occupied  
#  
> 💁 **o1lab**  
#### pls try getting rds to work with knex  
#  
> 💁 **o1lab**  
#### http://knexjs.org/guide/#configuration-options  
#  
> 🙋‍♂️ **ioakshay**  
#### Okay try if we get your 4 minutes of time  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1004687010503598121)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
